packages/generate-html/test/generate-html.spec.ts

```diff
@@ -112,7 +133,21 @@ describe('GenerateHTML', function () {
       partialsMap: {
         company: 'https://schemas.s1seven.dev/schema-definitions/v0.0.5/company/company.hbs',
       },
-      localOnly: false,
+    },
+    {
+      type: SupportedSchemas.COA,
+      version: 'v1.1.0',
+      certificatePath: `${__dirname}/../../../fixtures/CoA/v1.1.0/valid_cert.json`,
+      schemaTranslationsPath: `${__dirname}/../../../fixtures/CoA/v1.1.0/translations.json`,
+      schemaExtraTranslationsPath: `${__dirname}/../../../fixtures/CoA/v1.1.0/extra_translations.json`,
+      schemaInterface: readFileSync(`${__dirname}/../../../fixtures/CoA/v1.1.0/certificate.ts`, 'utf-8'),
+      localTemplatePath: `${__dirname}/../../../fixtures/CoA/v1.1.0/template.hbs`,
+      expectedHtmlFromHbs: readFileSync(`${__dirname}/../../../fixtures/CoA/v1.1.0/template_hbs.html`, 'utf-8'),
+      expectedHtmlFromMjml: '',
+      partialsMap: {
+        company: 'https://schemas.s1seven.dev/schema-definitions/v0.0.5/company/company.hbs',
```

Comment:
maybe https://schemas.s1seven.dev/schema-definitions/v0.0.7/company/company.hbs would be the correct version ?